src/main/java/uk/gov/pay/card/model/CardInformation.java

```diff
@@ -40,10 +34,7 @@ public CardInformation(String brand, String type, String label, Long min, Long m
         this.max = max;
         this.corporate = corporate;
         this.prepaidStatus = prepaidStatus;
-
-        if (brand != null) {
-            this.brand = brandMapping.getOrDefault(brand, brand.toLowerCase());
-        }
+        this.brand = brand == null ? null : brandMapping.getOrDefault(brand, brand.toLowerCase());
```

Comment:
I’m not really convinced this is clearer than using an `if` statement

Reply:
Me neither, but it did let me make the field `final` - is it worth it?

Reply:
I do like `final`. This would also work:

```java
if (brand != null) {
    this.brand = brandMapping.getOrDefault(brand, brand.toLowerCase());
} else {
    this.brand = null;
}
```

(A `final` field has to be assigned exactly once in the constructor but that doesn’t mean there can be only one assignment statement!)